packages/lodestar/src/network/gossip/validation/utils/hasValidAttestationSlot.ts

```diff
@@ -0,0 +1,13 @@
+import {IBeaconConfig} from "@chainsafe/lodestar-config";
+import {getCurrentSlot} from "@chainsafe/lodestar-beacon-state-transition";
+import {Slot} from "@chainsafe/lodestar-types";
+import {ATTESTATION_PROPAGATION_SLOT_RANGE, MAXIMUM_GOSSIP_CLOCK_DISPARITY} from "../../../../constants";
+
+export function hasValidAttestationSlot(config: IBeaconConfig, genesisTime: number, slot: Slot): boolean {
```

Comment:
I think this function have the following signature:
`(config: IBeaconConfig, currentSlot: Slot, slot: Slot): boolean`
and be called like `hasValidAttestationSlot(config, chain.clock.currentSlot, aggregate.data.slot)`